File: js/CLZString.js
var CLZString=function()
{
}
CLZString.CompressBase64=function(_str)
{
	return LZString.compressToBase64(_str);
}
CLZString.DecompressBase64=function(_str)
{
	return LZString.decompressFromBase64(_str);
}

File: java/CLZString.java
package cell;

import dal.LZString;

public class CLZString 
{
	static public String CompressBase64(String input)
	{
		return LZString.compressToBase64(input);
	}
	static public String DecompressBase64(String input)
	{
		return LZString.decompressFromBase64(input);
	}
}
